package.json

```diff
@@ -1,138 +1,68 @@
 {
-  "name": "ahc-analytics",
-  "version": "3.2.1",
-  "browserslist": [
-    "> 1%",
-    "last 2 versions",
-    "IE 11"
-  ],
+  "name": "ahc-analytics-vue",
+  "version": "0.1.0",
+  "private": true,
   "scripts": {
-    "ng": "ng",
-    "start": "per-env",
-    "ci-test": "npm run test",
-    "conventional-changelog": "conventional-changelog",
-    "pre-start": "per-env",
-    "pre-start:development": "echo 'starting'",
-    "pre-start:production": "npm run build:prod",
-    "start:development": "ng serve --host 0.0.0.0 --port $HMR_PORT  --poll 1000 --disableHostCheck --sourceMap=false",
-    "start:production": "echo 'done'",
-    "build": "ng build",
-    "build:prod:low-mem": "npm run build -- --prod --aot",
-    "build:prod": "node --max_old_space_size=8000 ./node_modules/@angular/cli/bin/ng build --prod --aot",
-    "test": "jest",
-    "test:coverage": "jest --coverage",
-    "test:watch": "jest --watch",
-    "format": "prettier ./src/**/*.{ts,js,html,scss,css} --write",
-    "format:ci": "prettier ./src/**/*.{ts,js,html,scss,css} --check",
-    "lint": "ng lint",
-    "lint:fix": "ng lint ahc-analytics --fix",
-    "lint:styles": "stylelint ./src/**/*.scss",
-    "lint:ci": "npm run lint && npm run lint:styles",
-    "pree2e": "webdriver-manager update --standalone false --gecko false",
-    "e2e": "ng e2e",
-    "docs": "compodoc -p src/tsconfig.app.json -d docs",
-    "docs:serve": "compodoc -p src/tsconfig.app.json -d docs -s",
-    "release:changelog": "npm run conventional-changelog -- -p angular -i CHANGELOG.md -s",
-    "postinstall": "ngcc --properties es2015 es5 browser module main --first-only --create-ivy-entry-points --tsconfig \"./src/tsconfig.app.json\""
+    "start": "vite dev --host 0.0.0.0 --strictPort --port ${HMR_PORT:-4173}",
+    "build": "run-p type-check \"build-only {@}\" --",
+    "preview": "vite preview --host 0.0.0.0 --strictPort --port ${HMR_PORT:-4173}",
+    "test:unit": "vitest",
+    "prepare": "cypress install",
+    "test:e2e": "start-server-and-test preview http://localhost:${HMR_PORT:-4173} 'cypress run --e2e --config baseUrl=http://localhost:${HMR_PORT:-4173}'",
+    "test:e2e:dev": "start-server-and-test start http://localhost:${HMR_PORT:-4173} 'cypress open --e2e --config baseUrl=http://localhost:${HMR_PORT:-4173} --browser electron'",
+    "build-only": "vite build",
```

Comment:
you may need to add something like this `--max_old_space_size=8000`, because the server memory might not be enough for the build.
Anyway, we can test it first and if it doesn't work we can add it later.